packages/nestjs-module/src/interceptor/providers/redis-interceptor.service.ts

```diff
@@ -0,0 +1,5 @@
+import { Injectable } from '@nestjs/common';
+import { AbstractInterceptorService } from './abstract-interceptor.service';
+
+@Injectable()
+export abstract class RedisInterceptorService extends AbstractInterceptorService {}
```

Comment:
Was there a reason for the new class the `nestjs-module` package?

Reply:
Sorry, I've removed it.